apps/website/pages/blog/index.tsx

```diff
@@ -38,9 +43,11 @@ function mapStorybookToPosts(data: { posts: StorybookPost[] }): Post[] {
         content: post.content ?? '',
         image,
         isFeatured: post.featured,
+        isPublished: post.isPublished,
```

Comment:
remove, no need this here. we already filtered unpublished posts before

Reply:
Requested changes have been made